runtime/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 authors = ['Joystream contributors']
 edition = '2018'
 name = 'joystream-node-runtime'
-# Follow convention: https://github.com/Joystream/substrate-runtime-joystream/issues/1
+# Follow convention: https://github.com/mnaamani/substrate-runtime-joystream/issues/1
```

Comment:
https://github.com/mnaamani/substrate-runtime-joystream/issues/1 points to an unrelated issue, I think the previous link was correct

Reply:
fixed in 9f683846b567326d01dd3ca37068f74751d1bc26